README.md

```diff
@@ -67,17 +76,18 @@ resource "aws_ami_from_instance" "example" {
 | `security_groups`            | []             | List of Security Group IDs allowed to connect to creating instance        | Yes |
 | `subnets`                    | []             | List of VPC Subnet IDs creating instance launched in                      | Yes |
 | `zone_id`                    | ``             | ID of the domain zone to use - is a result of tf_domain output            | Yes |
+| `associate_public_ip_address`| `true`         | Associate a public ip address with the creating instance. Boolean value   | No  |
 
 ## Outputs
 
-| Name                | Decription              |
-|:-------------------:|:-----------------------:|
-| `id`                | Disambiguated ID        |
-| `fqdn`              | Normalized name         |
-| `public_ip`         | Normalized namespace    |
-| `ssh_key_pair`      | Name of used AWS SSH key|
+| Name                | Decription                                                        |
+|:-------------------:|:-----------------------------------------------------------------:|
```

Comment:
I prefer `:----` to `:----:`.  Use `:---:` for default values. Use `:----` for everything else.

Reply:
This row is still wrong.